doc/reference/guidelines.rst

```diff
@@ -6,24 +6,31 @@
 Odoo Guidelines
 ===============
 
-This page introduce the new Odoo Coding Guidelines. These guidelines
-aim to improve the quality of the code (better readability of source,
-...) and  Odoo Apps. Indeed, proper code ought ease maintenance, aid
-debugging, lower complexity and promote reliability.
+This page introduce the new Odoo Coding Guidelines. These guidelines aim to improve the quality of the code (better readability of source, ...) and Odoo Apps. Indeed, proper code ought ease maintenance, aid debugging, lower complexity and promote reliability.
+
+These guidelines should be applied to every new module, and new developpment. These guidelines will be applied to old module **only** in case of code refactoring (migration to new API, big refactoring, ...).
 
 Module structure
 ================
 
 Directories
 -----------
-A module is organised in a few directory :
+A module is organised in some important directories. These directories aim to contain the business core of the module; having a look at them should make understand the purpose of the module.
```

Comment:
- remove "some"
- Those contain the business logic ...
- Just looking at them should help understanding the module purpose.